src/Header.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledNav = styled.nav`
+  border: 2px solid red;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px 80px 10px 80px;
+`;
+
+const StyledList = styled.ul`
+  display: flex;
+  list-style: none;
+`;
+
+const StyledListItem = styled.li`
+  margin-right: -1px;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  padding-left: 26px;
```

Comment:
Having the padding here increases the clickable area. In some situation this might be desired but here the user might click into the empty space and be sent to another page. Not dramatic here since the space is small. But something to be aware of

Reply:
I'd say setting `margin-left: 25px` to the `StyledListItem` component and removing the padding here is a good alternative